web/src/components/molecules/View/types.ts

```diff
@@ -0,0 +1,11 @@
+import { Condition, FieldSelector, NewItemSort } from "@reearth-cms/gql/graphql-client-api";
+
+export type View = {
+  id: string;
+  name: string;
+  modelId?: string;
+  projectId?: string;
```

Comment:
modelId and projectId are mandetory
```suggestion
  modelId: string;
  projectId: string;
```